src/app/shared/api-locator.service.ts

```diff
@@ -36,7 +36,7 @@ export class BaseApiLocatorService {
   }
 
   get realm(): string {
-    return this.envVars.get('realm') || "fabric8";
```

Comment:
I think we should rename `envVars` to `buildTimeEnvVars`. That would make more sense since `envVars` doesn't contain actual environment variable values.